mongo.md

```diff
@@ -10,25 +10,50 @@ Administering data requires administrative access to the database. This access i
 
 # Collections
 
-* `incidents`: this is the main incident database
+* `incidents`: this is the main incident collection that links incidents to  reports in a one-to-many relationship
+* `reports`: represents a particular report of an incident
 * `quickadd`: A collection of links without their associated content. This is to make it easier to capture lots of data that needs to subsequently get processed into full records.
-* `submissions`: Prospective full incidents before they are rotated into the `incidents` collection. These should have the complete `incidents` schema minus the elements that are granted when promoted to incident reports.
+* `submissions`: Prospective full reports before they are rotated into the `reports` collection. These should have the complete `reports` schema minus the elements that are granted when promoted.
 * `duplicates`: All the incident numbers that have been migrated to an earlier incident number.
 * `taxa`: Metadata describing the taxonomies of the classifications collection.
 * `classifications`: All the taxonomic classifications of incident records.
 
-# Incidents Collection Details
+## Incidents Collection Details
 
-Systems
+### Systems
 
 * `_id`: 5534b8c29cfd494a0103d45a # MongoDB database hash
 * `incident_id`: 1 # (int) The incrementing primary key for incidents, which are a collection of reports.
-* `ref_number`: 25 # (int) The reference number scoped to the incident ID.
+* `reports`: [1, 2, 3] # (Int[]) The list of reports by `report_number`, that belong to this Incident
+
+### Dates
+
+* `date`: 2019-07-25 # (String) Date the incident occurred.
+
+### People
+
+* `editors`: ["Anonymous"] # (String[]) Editors responsible for the Incident
+
+### Text
+
+* `title`: ["Anonymous"]: String title
+
+### Other
+
+* `Alleged deployer of AI system`: String[] ??
+* `Alleged developer of AI system`: String[] ??
```

Comment:
`Alleged developer of AI system`: String[] The entities responsible for developing the AI system in the real world.